src/koans/struct.rs

```diff
@@ -0,0 +1,74 @@
+// Structs are a convenient way to wrap up related data into one neatly packaged structure.
+// Let's consider the example of a Struct that allows us to package together data about a person.
+#[test]
+fn our_first_struct() {
+  struct Person {
+    name: &'static str,
+    age: u32
+  }
+
+  let jim = Person { name: "Jim", age: 57 };
+
+  assert_eq!(jim.name, __);
+  assert_eq!(jim.age, __);
+}
+
+// Let's try another example
+#[test]
+  let movie = Movie { title: "Star Wars", runtime: 121 };
+
+  assert_eq!(movie.title, "Star Wars");
+  assert_eq!(movie.runtime, 121);
+}
+
+// Structs, like anything in Rust, are immutable by default.
+// If we create a mutable instance of a Struct, we can reassign its attributes.
+#[test]
+fn mutable_structs() {
+  struct Language {
+    version: f32
```

Comment:
Using floats to store version numbers is Bad News Bears. Even if it doesn't matter here, it either makes us look ignorant or encourages the dungus.